packages/aws-cdk-lib/aws-lambda/README.md

```diff
@@ -1143,6 +1143,28 @@ const fn = new lambda.Function(this, 'Lambda_with_IPv6_VPC', {
 });
 ```
 
+## Outbound traffic
+By default, when creating a Lambda function, it would add a security group outbound rule to allow sending all network traffic (except IPv6). This is controlled by `allowAllOutbound` in function properties, which has a default value of `true`.
+
+To allow outbound IPv6 traffic by default, explicitly set `allowAllIpv6Outbound` to `true` in function properties as shown below (the default value for `allowAllIpv6Outbound` is `false`):
+```ts
+import * as ec2 from 'aws-cdk-lib/aws-ec2';
+
+const vpc = new ec2.Vpc(this, 'Vpc', {
+  maxAzs: 2,
+});
```

Comment:
```suggestion
const vpc = new ec2.Vpc(this, 'Vpc');
```

Let's just remove optional props and keep the sample as simple as possible.

Reply:
Done